src/compas_rhino/geometry/sphere.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+import Rhino
+import compas_rhino
+from compas.geometry import Sphere
+from ..conversions import sphere_to_rhino
+from ..conversions import sphere_to_compas
+from ..conversions import ConversionError
+from ._geometry import BaseRhinoGeometry
+
+
+class RhinoSphere(BaseRhinoGeometry):
+    """Wrapper for Rhino sphere objects.
+    """
+
+    @classmethod
+    def from_object(cls, obj):
+        """Construct a sphere wrapper from an existing Rhino object.
+
+        Parameters
+        ----------
+        obj : :class:`Rhino.DocObjects.ExtrusionObject`
+            The Rhino object.
+
+        Returns
+        -------
+        :class:`RhinoSphere`
+            The Rhino sphere wrapper.
+
+        Raises
+        ------
+        :class:`ConversionError`
+            If the Rhino object cannot be converted to a sphere.
+        """
+        wrapper = cls()
+        wrapper.guid = obj.Id
+        wrapper.object = obj
+        geometry = obj.Geometry
+        if not isinstance(geometry, Rhino.Geometry.Sphere):
+            if isinstance(geometry, Rhino.Geometry.Brep):
+                if geometry.Faces.Count != 1:
+                    raise ConversionError('Object brep cannot be converted to a sphere.')
+                face = geometry.Faces.Item[0]
+                if not face.IsSphere():
+                    raise ConversionError('Object brep cannot be converted to a sphere.')
+                result, geometry = face.TryGetSphere()
+                if not result:
+                    raise ConversionError('Object brep cannot be converted to a sphere.')
+            else:
+                raise ConversionError('Rhino object cannot be converted to a sphere: {}'.format(obj))
+        wrapper.geometry = geometry
```

Comment:
these blocks (present in several of the new methods/classes feel like they could become the `coerce###` stuff I just mentioned.... right?

Reply:
yes